cmd/kubeadm/app/preflight/checks_linux.go

```diff
@@ -51,8 +51,8 @@ func (mc MemCheck) Check() (warnings, errorList []error) {
 		errorList = append(errorList, errors.Wrapf(err, "failed to get system info"))
 	}
 
-	// Totalram returns bytes; convert to MB
-	actual := uint64(info.Totalram) / 1024 / 1024
+	// Totalram returns digits; Unit returns bit size a single digit of Totalram represents; Totalram * Unit = bytes; convert to MB
```

Comment:
```suggestion
	// Totalram holds the total usable memory. Unit holds the size of a memory unit in bytes. Multiply them and convert to MB
```
^ suggesting a minor adjustment to the text. please keep the commits squashed to one.

Reply:
Made this edit -- tbh have not squahsed two commits together before and it keeps just making a new commit when I squash locally and push. If this is OK I will leave as is

Reply:
@neolit123 -- finally got it 👍